test/python/api_config.py

```diff
@@ -61,6 +61,14 @@ def get_api_client(username='admin'):
     client.configuration.api_key = {'Authorization': f'Token token="{api_token}"'}
     return client
 
+def load_default_policy(account, client):
```

Comment:
because this just uses two class variables it should probably be moved into `ConfiguredTest` as an `@classmethod` with no parameters